### DISCUSSION TITLE
Is it possible to change the logo?

### DISCUSSION THREAD

**Discussion by Animeshz** [NoiSek/Aether#101] [Q&A] [answered]
Hi! I'm using void linux, is it possible to put Void logo instead of Arch logo?

**Comment by Animeshz** [comment]
@NoiSek 👀

**Comment by NoiSek** [comment, marked as answer]
The logos are generated automatically from whatever files are present in the logos directory: https://github.com/NoiSek/Aether/blob/master/src/js/Logic/FileOperations.js#L55

Drop an appropriate logo file with a transparent background and you should see it pop up automatically under the General Settings.

**Reply by Animeshz** [reply]
Oh, can you please add that to the readme, its not documented anywhere (or maybe can't be found info easily)?
